rest/src/main/kotlin/systems/ajax/englishstudytelegrambot/nats/controller/library/CreateNewLibraryNatsController.kt

```diff
@@ -27,13 +27,13 @@ class CreateNewLibraryNatsController(private val libraryService: LibraryService)
             createFailureResponse(it)
         }
 
-    private fun createdLibrary(request: CreateNewLibraryRequest): LibraryOuterClass.Library {
+    private fun createdLibrary(request: CreateNewLibraryRequest): Library {
         val createdLibrary =
             libraryService.createNewLibrary(request.libraryName, request.telegramUserId).toLibraryResponse()
         return createdLibrary
     }
```

Comment:
If you use a kotlin-like code style in your class, it is better to stick to it in all functions

```kotlin
 private fun createdLibrary(request: CreateNewLibraryRequest): Library =
            libraryService.createNewLibrary(request.libraryName, request.telegramUserId).toLibraryResponse()
```